README.md

```diff
@@ -1,70 +1,247 @@
+                </a>
+                <br><b>Supply Chains</b>
+            </td>
+            <td align="center">
+                <a href="https://dkg.origintrail.io/explore?ual=did:dkg:otp/0x5cac41237127f94c2d21dae0b14bfefa99880630/309285">
+                  <img src="images/knowledge-assets-graph2.svg" width="300" alt="Knowledge Assets Graph 2">
+                </a>
+                <br><b>Construction</b>
+            </td>
+            <td align="center">
+                <a href="https://dkg.origintrail.io/explore?ual=did:dkg:otp/0x5cac41237127f94c2d21dae0b14bfefa99880630/309222">
+                  <img src="images/knowledge-assets-graph3.svg" width="300" alt="Knowledge Assets Graph 3">
+                </a>
+                <br><b>Life sciences and healthcare</b>
+            </td>
+            <td align="center">
+                <a href="https://dkg.origintrail.io/explore?ual=did:dkg:otp/0x5cac41237127f94c2d21dae0b14bfefa99880630/308028">
+                  <img src="images/knowledge-assets-graph4.svg" width="300" alt="Knowledge Assets Graph 3">
+                </a>
+                <br><b>Metaverse</b>
+            </td>
+        </tr>
+    </table>
+</div>
 
-* Enables Dapps with search, integration, analytics, AI and ML capabilities for any data source: blockchains, IPFS, enterprise systems, web services, personal devices 
-* Removes central authorities (decentralized infrastructure)
-* Enables permissionless PUBLISH and QUERY (public network)
-* Decentralized identity & Verifiable Credentials based access control (references private data)
 
-## The OriginTrail DKG Architecture 
+</details>
 
-The OriginTrail Decentralized Network implements the DKG according to the OriginTrail protocol.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<br/>
 
-It is:
+## 🚀 Getting Started
 
-* **a permissionless network** - anyone can run OriginTrail nodes
-* **a multi-chain data exchange network** - connects to several blockchains (currently Ethereum and xDai with more integrations upcoming such as with Polkadot)
-* **designed for off-chain data exchange using standardized data models** (GS1 & W3C standards and recommendations)
-* **public open source software**
-* **infrastructure for knowledge marketplaces & tenders** - more info [here](https://www.youtube.com/watch?v=4uCxYGRh5fk)
+___
 
-More information is available on the OriginTrail [website](https://origintrail.io), [official documentation](https://docs.origintrail.io) and [blog](https://medium.com/origintrail).
+### Prerequisites
 
+- **Node.js** >= 16.0.0
+- **npm** >= 8.0.0
 
-![](https://i.imgur.com/yTNtZE1.png)
+___
+<br/>
 
+### Installation
 
+<br/>
 
-## DKG Client library
+Install the SDK:
+```bash
+npm install dkg.js
+```
 
-This library provides an interface into the OriginTrail Decentralized Knowledge Graph, enabling:
+Now you can run a demo example file: 
```

Comment:
I would expand here that is prerequisite to have a running local dkg network, also to take a look at other examples